src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -241,8 +241,8 @@
                             <Border x:Name="ControlPreview"
                                     Width="350"
                                     Height="160"
-                                    BorderBrush="#333333"
-                                    BorderThickness="1" />
+                                    BorderThickness="1"
+                                    BorderBrush="{ThemeResource ToggleButtonDisabledBorderThemeBrush}"/>

```

Comment:
As long as this.. uh, looks fine. I guess. It's a weird one to use-- we're not a toggle button :smile:

Reply:
This is an older commit! It uses `SystemControlForegroundBaseMediumLowBrush`